groups-domains.php

```diff
@@ -88,6 +88,11 @@
                         </div>
                     </div>
                 </div>
+                <div>
+                    <p><strong>Note:</strong><br>
+                    The domain or regex filter will be assigned to the Default Group only.<br>
+                    Other groups will need to be selected within <a href="groups-domains.php">Group Management > Domains</a>.</p>
```

Comment:
```suggestion
                    Other groups can optionally be assigned within <a href="groups-domains.php">Group Management > Domains</a>.</p>
```

Just playing about with the copy here a little - not fully settled on this, but I think it currently reads a little like it is a mandatory process.

Reply:
I agree.
Setting other groups are optional.

Reply:
Agree, good point.